docs/contributing/coding-standards/components.md

```diff
@@ -21,7 +21,8 @@ When creating your component, you should create the following files in the compo
 If your component uses JavaScript, you must also create the following files in the component’s folder:
 
       - `_[component-name].mjs`
-      - `_[component-name].test.js`
+      - `[component-name].unit.test.mjs`
```

Comment:
Just checking that the underscores aren't needed before the square brackets here.

Reply:
The underscores aren't needed - we don't use underscores in these filenames. I've actually just realised we don't use them in the `.mjs` files either (the top bullet), so I'll remove that too!